### DISCUSSION TITLE
Freeplane_Tutorial_AddOn v0.0.7

### DISCUSSION THREAD

**Discussion by EdoFro** [freeplane/docs#68] [Show and tell]
Hi all!
I have been working again in the tutorial add-on.

Here is the link to the new [release](https://github.com/EdoFro/Freeplane_Tutorial_AddOn/releases/tag/v0.0.7)
and the link to its [repository](https://github.com/EdoFro/Freeplane_Tutorial_AddOn)

Please, install it and execute the menu command `Edo Tools->Tutorial-o-Matic->Tutorial-o-Matic Intro`

I need your opinions.

I personally think it can be useful to create some basic scripting tutorials.

bye,
edo

**Comment by macmarrum** [comment]
@EdoFro I've installed it and I am planning to have a deeper look.
Kudos to you for the continued development of the add-on :star: 
It has a great potential :rocket:

**Comment by EdoFro** [comment]
I added a new command to the addon. Just as a test:

It opens a help tab for the "asked" menu command

## getMenuHelp.groovy (Control + Shift + F1)

For the test I created a map with just a few menu commands: Test only works with the commands inside "**View / Notes /** ".

To repeat the test:

- open the menu "View / Notes"
- point with the mouse arrow on a command from that submenu (without clicking)
- press **Control + Shift + F1**

here is a gif from the test:
![ToM_getMenuHelp](https://user-images.githubusercontent.com/35700575/184516219-438e907a-a6f8-46dd-a939-4d965f714ac6.gif)

When executed it :

1. "sees" if the mouse arrow is over a menu command and reads its related `action.key`
2. opens the mindmap with the helping information (hidden)
3. looks in the map a node with a link to the same command
4. it opens the tutorial page that contains that node

You can download it from its [release page](https://github.com/EdoFro/Freeplane_Tutorial_AddOn/releases/tag/v0.0.8)

bye,

Edo

**Reply by macmarrum** [reply]
Nice :+1:
This can be really useful.
So now there is a way to go both ways: from the docs/tutorial to a menu item, and from a menu item to docs :smile:
